scipy/optimize/_differentialevolution.py

```diff
@@ -486,6 +502,11 @@ class DifferentialEvolutionSolver:
         Provides an initial guess to the minimization. Once the population has
         been initialized this vector replaces the first (best) member. This
         replacement is done even if `init` is given an initial population.
+    integrality : 1-D array, optional
+        For each decision variable, a boolean value indicating whether the
+        decision variable is constrained to integer values. If any decision
+        variables are constrained to be integral, polishing will not be
+        performed (e.g. ``polish=False``).
```

Comment:
```suggestion
        decision variable is constrained to integer values.
```
Maybe note in the `finish` section that 

>Decision variables constrained to be integral will not be refined by polishing.

Reply:
ACK